sonic_data_client/db_client.go

```diff
@@ -717,6 +717,10 @@ func tableData2Msi(tblPath *tablePath, useKey bool, op *string, msi *map[string]
 			var key string
 			// Split dbkey string into two parts and second part is key in table
 			keys := strings.SplitN(dbkey, tblPath.delimitor, 2)
+			if len(keys) < 2 {
+				log.V(3).Infof("dbkey: %s, failed split from delimitor %v", dbkey, tblPath.delimitor)
```

Comment:
This line adds to log spam. Is it _really_ necessary?

Reply:
We don't expect to go into this if block, very rare if it does happen. If it does happen, we want to know via log

Reply:
The problem is that if the condition persists then the log will be printed out every time this function is called. 

This function returns the same message as part of the error status. Shouldn't the caller make the decision if the error message should be printed?

Reply:
Yes, I will remove as we are returning error which will be handled by caller